x-pack/plugins/enterprise_search/public/applications/workplace_search/views/account_settings/account_settings.test.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import '../../../__mocks__/shallow_useeffect.mock';
+import { mockKibanaValues } from '../../../__mocks__/kea_logic';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { AccountSettings } from './';
+
+describe('AccountSettings', () => {
```

Comment:
@yakhinvadim Thought I'd write some unit tests for you while I was here! LMK if you have any questions on the file - wasn't too bad or anything, just overriding the mocks provided by the security team

Reply:
OMG thank you so much! 🙇 
TIL I can do `await shallow`. I was struggling with trying to make nextTick() work.

Reply:
Haha yeah the tests were totally blowing up with `nextTick()` for some reason, it was strange. Also my secret to async testing is I sticking `await` in front of random things until tests start working :trollface:

Reply:
🤣